docker-compose.yml

```diff
@@ -29,4 +46,4 @@ services:
       - CSP_REPORT_ENABLE=False
```

Comment:
`web` has a bunch of env variables set originally and you only leave it with 3. Copy / paste erorr?

Reply:
good catch, thanks! I've restored the accidentally deleted env vars in ad0ff18

Reply:
Thanks for updating. I belive `ENABLE_ADMIN` and the `CSP` variables are not needed to run Jupiter. Let's try to keep things tidy!

Reply:
thanks, removed in b64932c